RetrievingAndDeleting.md

```diff
@@ -30,28 +33,52 @@ The following code can be added to the FoodTracker app and will query the saved
         }
     }
 ```  
-Try inserting this into the FoodTracker app and loading the Meals from the FoodServer.
+The above code will query the saved Meals from the server, and update the TableView.
 
-### Deleting Meals from the FoodServer  
-The design of the FoodTracker app is such that it saves all of the Meals when any Meal is added or removed, overwriting the previous store of Meals. Whilst this provides an implementation, it is not one that scales to large numbers of Meals.
+4. Call this new function by replacing the `loadMeals` function with:
+```swift
+private func loadMeals() -> [Meal]?  {
+        loadFromServer()
+        return meals
+    }
+```
+5. Inside your `saveMeals` function, replace:
+```swift
+do {
+    let data = try PropertyListEncoder().encode(meals)
+    let isSuccessfulSave  = NSKeyedArchiver.archiveRootObject(data, toFile: MealTableViewController.ArchiveURL.path)
```

Comment:
This is fine for now, but perhaps setting up Core Data, or even using something like Realm, might be better in the future 😄 just for the sake of "best practices" on the iOS side. This is an old school way of doing it.